src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,42 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_LENGTH = 6;
```

Comment:
The constant name 'MINIMUM_LENGTH' could be more descriptive. According to the checklist, constant names should be informative. Consider renaming it to 'MIN_PASSWORD_LENGTH' to clarify what this minimum length refers to.